ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature

```diff
@@ -0,0 +1,112 @@
+    Then check visibility of page "Review Actions"
+
+    When click on the "Next" link
+    Then check visibility of page "Specify Security"
+
+    And click on the "Next" button
+    And fill in values by element data-testid
+      | name     | Todo connector |
+      | basepath | /api           |
+    And fill in TODO API host URL
+    And click on the "Save" button
+
+    When created connections
+      | Todo connector | todo | Todo connection | no validation |
+    And navigate to the "Home" page
+
+    When create an API Provider integration "TODO Integration get single" from file swagger/connectors/todo.json
+    And select API Provider operation flow Fetch task
+    Then check flow title is "Fetch task"
+
+    When add integration step on position "0"
+
+    When select the "Todo connection" connection
+    Then select "Create new task" integration action
+    And click on the "Next" button
+
+    When add integration step on position "0"
+
+    And select "Data Mapper" integration step
+    And create data mapper mappings
+      | parameters.id | body.id |
+    And sleep for jenkins delay or "2" seconds
+    And check "Done" button is "visible"
+    And click on the "Done" button
+
+    And click on the "Save" link
+    And publish integration
+    And navigate to the "Integrations" page
+    Then wait until integration "TODO Integration get single" gets into "Running" state
+
+    When execute GET on API Provider route i-todo-integration-get-single endpoint "/api/1"
+    Then verify response has status 200
+    And verify response has body
+        """
+        {"id":"1","task":null,"completed":null}
+        """
+    And select the "TODO Integration get single" integration
+    Then check visibility of "TODO Integration get single" integration details
+    When clean webdriver download folder
+    And export the integraion
+
+    # now we have exported integration, we can clean state and try to import
+    And clean application state
+    And log into the Syndesis
+    And navigate to the "Integrations" page
+    And click on the "Import" link
+    Then import integration "TODO Integration get single"
+
+    When navigate to the "Integrations" page
+    Then Integration "TODO Integration get single" is present in integrations list
+    Then wait until integration "TODO Integration get single" gets into "Stopped" state
+
+    When select the "TODO Integration get single" integration
+    And check visibility of "Stopped" integration status on Integration Detail page
+    And sleep for jenkins delay or "3" seconds
+
+    And click on the "Edit Integration" link
+    And click on the "Save" link
+    And publish integration
+    And navigate to the "Integrations" page
+    Then Integration "TODO Integration get single" is present in integrations list
+
+    Then wait until integration "TODO Integration get single" gets into "Running" state
+
+    When execute GET on API Provider route i-todo-integration-get-single endpoint "/api/1"
```

Comment:
The /api/1magically produces task with id 2, this doesn't look like the intended behavior, for sake of future, please rewrite this to make more sense.